gatsby-config.js

```diff
@@ -115,6 +125,7 @@ module.exports = {
         accessToken: process.env.GATSBY_STORYBLOK_ACCESS_TOKEN,
         homeSlug: "home",
         resolveRelations: storyblokRelations,
+        resolveLinks: 'url',
```

Comment:
While looking at the netlify preview, I found that it has that same weird /default in the Main nav link issue we had on alumni, so I added this bit here, but I'm still seeing that problem on the Netlify preview. @mattanglin or @sherakama do you remember what else we had to do to fix that problem? I believe Nathan also mentioned that he ran into this issue as well.

Reply:
@yvonnetangsu To fix this we need to manually add the `storyblok-js-client@4.1.3` (locked to the version) to our package.json. It's a dependency of the `gatsby-source-storyblok` package, but they published some breaking changes in v4.1.4...

Reply:
Ahh that was it. Will do. Thanks, @mattanglin !